Library/Homebrew/cmd/info.rb

```diff
@@ -67,19 +68,36 @@ class Info < AbstractCommand
                description: "Treat all named arguments as formulae."
         switch "--cask", "--casks",
                description: "Treat all named arguments as casks."
+        switch "--leaves",
+               description: "Only show formulae and casks that are not dependencies of other " \
+                            "installed formulae/casks."
```

Comment:
Wondering if we even need this flag, since the following should result in equivalent output:
```sh
brew leaves | xargs brew info --sizes
```